samples/cpp/benchmark_app/main.cpp

```diff
@@ -1094,9 +1094,9 @@ int main(int argc, char* argv[]) {
 
         if (!FLAGS_exec_graph_path.empty()) {
             try {
-                std::string fileName = fileNameNoExt(FLAGS_exec_graph_path);
-                ov::pass::Serialize serializer(fileName + ".xml", fileName + ".bin");
-                serializer.run_on_model(std::const_pointer_cast<ov::Model>(compiledModel.get_runtime_model()));
+                ov::serialize(std::const_pointer_cast<ov::Model>(compiledModel.get_runtime_model()),
```

Comment:
We shouldn't use const_pointer_cast. Can we provide a helper and use const_cast on our side?

Reply:
Done.